src/v2/Apps/Order/Components/PaymentMethodSummaryItem.tsx

```diff
@@ -11,10 +11,12 @@ import { WireTransferDetails } from "./WireTransferDetails"
 export const PaymentMethodSummaryItem = ({
   order: { creditCard, paymentMethod },
   textColor = "black100",
+  currentStep,
```

Comment:
non-blocking: I would prefer not to send information about the step to the component and use a boolean field instead. Unfortunately, I don't have a good name for that field but maybe `compact` or `withDescription` will be fine.

Reply:
good idea!